acquire_zarr/launch/oryx_to_zarr.launch.py

```diff
@@ -0,0 +1,62 @@
+from launch_ros.substitutions import FindPackageShare
+
+from launch import LaunchDescription
+from launch_ros.actions import Node
+from launch.actions import OpaqueFunction, IncludeLaunchDescription, DeclareLaunchArgument as LaunchArg
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import PathJoinSubstitution, LaunchConfiguration as LaunchConfig
+
+oryx_parameters = {
+    'debug': False,
+    'compute_brightness': False,
+    'adjust_timestamp': True,
+    'dump_node_map': False,
+    'frame_rate_auto': 'Off',
+    'buffer_queue_size': 10,
+}
+
+def launch_setup(context, *args, **kwargs):
+    """Launch camera driver node."""
+    parameter_file = PathJoinSubstitution(
+            [FindPackageShare('spinnaker_camera_driver'), 'config', 'oryx.yaml']
+        )
+    
+    camera_node = Node(
+        package='spinnaker_camera_driver',
+        executable='camera_driver_node',
+        output='screen',
+        name='oryx',
+        parameters=[
+            oryx_parameters,
+            {
+                'ffmpeg_image_transport.encoding': 'hevc_nvenc',
```

Comment:
I'm curious why this was added. For scientific data, we can't be streaming w/ lossy compression.